### DISCUSSION TITLE
Templating - how to get "tomorrow"?

### DISCUSSION THREAD

**Discussion by chechoochoo** [mockoon/mockoon#936] [Help] [answered]
👋  could someone advise on how to retrieve "tomorrow" using templates? Tried combination of `dateTimeShift` and `now` but cannot figure out how to achieve this 😭 

Sample I would expect to work:
```json
{
    "dateTime": "{{dateTimeShift date='{{now}}' days=1}}"
}
```

however I get _Error while serving the content: Invalid time value_

**Comment by chechoochoo** [comment, marked as answer]
Ok, found a solution eventually 😝 

```
{
    "xxx": "{{dateTimeShift date=(now) days=1}}"
}

```

**Reply by 255kb** [reply]
Indeed, you cannot imbricate curly braces in Handlebars 🙂
